s2wav/Dimension helper functions.py

```diff
@@ -0,0 +1,370 @@
+# This file translates functions which can be called to check dimensions of vectors from s2let (c).
+
+from cmath import log
+from functools import total_ordering
+import math
+from tkinter import N
+
+
+#Defining functions originally called from ssht
+
+
+def ssht_sampling_mw_ss_nphi(L: int) -> int:
+    '''Computes the number of phi samples for McEwen and Wiaux symmetric sampling.
+    
+    Args:
+        L: Upper harmonic band-limit.
```

Comment:
Should include types whenever possible (as you've done in the function arguments). So this would read

`L (int): Upper harmonic band-limit.`

We should make sure this is consistent throughout.